src/goods.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+type Props = {
+  goods: string[];
+};
+
+export const GoodList: React.FC<Props> = ({ goods }) => {
```

Comment:
The name of the component 'GoodList' does not start with a verb, which is recommended for function names to clarify their behavior. Consider renaming it to something like 'RenderGoodList' to better convey the action the function performs.